src/js/site.js

```diff
@@ -86,5 +90,75 @@ jquery.fn.sui_module = function () {
   });
 };
 
+/**
+ * Plausible tracking module
+ *
+ * This reuses jQuery to provide explicit tracking of events at Plausible. To
+ * use events, add the ``data-analytics`` attribute to an element. In most
+ * cases, this should be a link element, however in the case of other UI
+ * components, it may be a ``<div>`` or ``<button>``:
+ *
+ *     <button class="ui button" data-analytics="some-event-id">Something</button>
+ *
+ * In the case of a link with a ``href`` attribute, the link will continue
+ * redirecting after the callback from Plausible fires off.
+ */
+jquery.fn.plausible = function () {
+  let plausible_settings = { domain: "about.readthedocs.org" };
```

Comment:
```suggestion
  let plausible_settings = { domain: "about.readthedocs.com" };
```

Reply:
We need to update this in plausible. I can do that today.

Reply:
Updated.